docs/blog/2022-08-12-observing-randomness.md

```diff
@@ -0,0 +1,140 @@
+
+**Minimize the need for opening network ports**
+
+Since drand nodes are run by multiple organizations on their own networks, any changes to the firewall configuration increases friction and reduces the probability of success. Ideally no network ports other than the one used for generating randomness should need to be opened. At the very least no more incoming ports should be needed.
+
+**Use industry-standard formats where possible**
+
+To decrease the ramp-up time for integrating into an organization’s monitoring system, we need to use an industry-standard format to publish metrics.
+
+**Ability to see the state of the whole network from any point in the network**
+
+To allow all LoE members to create their own dashboards and alerts, we need to enable each node to provide a full view of the whole network.
+
+**(Close to) real-time view of the network state**
+
+Especially during ceremonies, it is important to see changes in node state as soon as possible (< 1 minute) after they happen.
+
+## Design decisions
+
+The above requirements led us to the following design:
+
+**Each node exposes its own metrics in Prometheus format**
+
+Prometheus is an industry-standard format, and each node exposing its own metrics allows anyone to monitor their own node directly.
+
+**Additionally, each node is able to retrieve any other connected node’s metrics and expose them in Prometheus format**
+
+This enables not only Protocol Labs but any LoE member to create their own dashboards which show the state of the whole network.
+
+**The transport of metrics between nodes is done using the same mechanism that drand uses for inter-node communication when generating randomness (GRPC)**
+
+With this we allow metrics to be sent between nodes, while not requiring opening any additional network ports.
+
+**For the Protocol Labs dashboard, gather the metrics using Telegraf and export them to InfluxDB, which we can access via Grafana Cloud**
+
+This enables us to gather metrics from the whole swarm into a central location and create an easy-to-use dashboard. The scraping interval is set to one minute, and it can be lowered during ceremonies in order to get faster network state updates.
+
+## Metrics design
+
+To start with, we had to decide what we needed to monitor on each node. We came up with the following minimal information requirements for running a smooth ceremony:
+
+- Version of drand that the node is running, in order to ensure that all nodes are running the same/correct version.
+- Whether nodes have restarted their daemon ahead of the ceremony, to ensure that the certificate is still valid and to clear the process’ memory and storage.
+- Whether nodes have connectivity to all other nodes.
+- When and if the node started the ceremony process.
+- What is the current state of the node related to the ceremony.
```

Comment:
`what the current state of the node is with relation to the ceremony` or perhaps just `the current ceremony status of the node`

Reply:
Albeit is there a difference between `When and if the node started the ceremony process` and `What is the current state of the node related to the ceremony`? They seem to be overlapping